primitives/runtime-interface/proc-macro/src/pass_by/inner.rs

```diff
@@ -52,7 +52,7 @@ pub fn derive_impl(mut input: DeriveInput) -> Result<TokenStream> {
 			#crate_include
 
 			impl #impl_generics #crate_::pass_by::PassBy for #ident #ty_generics #where_clause {
-				type PassBy = #crate_::pass_by::Inner<#ident, #inner_ty>;
+				type PassBy = #crate_::pass_by::Inner<#ident #ty_generics, #inner_ty>;
```

Comment:
```suggestion
				type PassBy = #crate_::pass_by::Inner<Self, #inner_ty>;
```

Probably much easier to just use `Self` :D